sdk/python/tests/compiler/testdata/pipelineparams.yaml

```diff
@@ -45,48 +45,43 @@ spec:
       default: latest
     tasks:
     - name: download
-      params:
-      - name: sleep_ms
-        value: $(params.sleep_ms)
-      - name: tag
-        value: $(params.tag)
       taskSpec:
         steps:
         - name: main
           args:
-          - sleep $(inputs.params.sleep_ms); wget localhost:5678 -O $(results.downloaded-resultoutput.path)
+          - |
+            sleep None; wget localhost:5678 -O $0
```

Comment:
it should be `sleep $(inputs.params.sleep_ms);`. Do you think valuable replacement for image works in component.yaml?

Reply:
I guess it doesn't work. then we need to fix the compiler error. let me revert this and put a comment on issue 670